sycl/include/sycl/platform.hpp

```diff
@@ -57,6 +62,40 @@ namespace ext::oneapi {
 class filter_selector;
 } // namespace ext::oneapi
 
+#ifdef __INTEL_PREVIEW_BREAKING_CHANGES
+template <typename ParamT, typename ReturnT, typename PropT>
+ReturnT _get_info_impl(ParamT &Info) {
```

Comment:
What is this leading underscore? We don't use that convention in our code. Put it into the `sycl::detail` namespace and name something like `convert_to_abi_netural` and `convert_from_abi_neutral`. Also, why do you need three type parameters? Can't you deduce them here?

Reply:
Sure, I changed as you suggested.
Thanks.